src/commcare_cloud/terraform/modules/commcarehq/main.tf

```diff
@@ -24,15 +25,18 @@ module "network" {
 
 locals {
   subnet_options = {
-    app-private-a = "${module.network.subnet-a-app-private}"
-    app-private-b = "${module.network.subnet-b-app-private}"
-    app-private-c = "${module.network.subnet-c-app-private}"
-    db-private-a = "${module.network.subnet-a-db-private}"
-    db-private-b = "${module.network.subnet-b-db-private}"
-    db-private-c = "${module.network.subnet-c-db-private}"
-    public-a = "${module.network.subnet-a-public}"
-    public-b = "${module.network.subnet-b-public}"
-    public-c = "${module.network.subnet-c-public}"
+    app-private-a = "${lookup(module.network.subnets-app-private, "a", "")}"
+    app-private-b = "${lookup(module.network.subnets-app-private, "b", "")}"
+    app-private-c = "${lookup(module.network.subnets-app-private, "c", "")}"
+    app-private-f = "${lookup(module.network.subnets-app-private, "f", "")}"
+    db-private-a = "${lookup(module.network.subnets-db-private, "a", "")}"
+    db-private-b = "${lookup(module.network.subnets-db-private, "b", "")}"
+    db-private-c = "${lookup(module.network.subnets-db-private, "c", "")}"
+    db-private-f = "${lookup(module.network.subnets-db-private, "f", "")}"
+    public-a = "${lookup(module.network.subnets-public, "a", "")}"
+    public-b = "${lookup(module.network.subnets-public, "b", "")}"
+    public-c = "${lookup(module.network.subnets-public, "c", "")}"
+    public-f = "${lookup(module.network.subnets-public, "f", "")}"
```

Comment:
As an example of accepting a new zone, 'f', I added new lines here. It'd be nice, and I _think_ possible but annoying to do in terraform, to have this map get computed from the other variables (like `az_codes`), but this works, is easy to add to if we need to use it in a pinch, and is still more flexible than it was before at least, requiring _less_ of a change to support a new AZ.